File: 283-move-zeroes/move-zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        j = 0
        if len(nums) == 1:
            print(nums)
        else:
            for i in range(0, len(nums)):
                if nums[i] != 0:
                    nums[j] = nums[i]
                    j += 1
            
            while j < len(nums):
                nums[j] = 0
                j += 1

File: 283-move-zeroes/move-zeroes.c
void moveZeroes(int* nums, int numsSize) {
    
    if(numsSize == 1){
        printf("%d", nums[0]);
    }
    else{
        int count = 0, j=0, i;
        for(i=0; i<numsSize; i++){
            if(nums[i] != 0){
                nums[j++] = nums[i];
            }
        }
        while(j < numsSize){
            nums[j++] = 0;
        }
    }
}